docker-compose.yaml

```diff
@@ -13,12 +13,18 @@ services:
       - ./frontend:/app
     stdin_open: true
     tty: true
+    environment:
+      - NEXTAUTH_URL=http://localhost:3000/api/auth
+      - NEXTAUTH_SECRET=wbb33dOF5jxX1s8Y7gXhEdIgD/zCzwRt659+0YIEgEo=
+      - GITHUB_ID=4e29058a74f57c9fd62a
+      - GITHUB_SECRET=18688aafe7221621dc420439cfe1ca753b9fb5c6
+
 
   collaboration-service:
     build: ./collaboration-service
     container_name: collaboration_service_container
     ports:
-      - "3001:3001"
+      - "3001:3000"
```

Comment:
3000 is already exposed in frontend, not sure if this mapping will cause a conflict of ports with collab service

Reply:
Liveblocks uses 3000 as port, so I have to use 3000 as tcp and our local host for whiteboard is 3001.